frontend/coreproject/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,156 @@
+<script lang="ts">
+	export let anime: any;
+
+	import ScrollArea from '$components/shared/ScrollArea.svelte';
+	import SidebarDetails from '$components/pages/anime_info/SidebarDetails.svelte';
+
+	// icons
+	import PlayCircle from '$icons/PlayCircle.svelte';
+	import Read from '$icons/Read.svelte';
+	import Listen from '$icons/Listen.svelte';
+	import Video from '$icons/Video.svelte';
+	import Download from '$icons/Download.svelte';
+	import Share from '$icons/Share.svelte';
+	import SettingsOutline from '$icons/SettingsOutline.svelte';
+	import Edit from '$icons/Edit.svelte';
+</script>
+
+<div class="grid grid-cols-12 items-start">
+	<div class="col-span-10 grid grid-cols-12 items-end">
+		<div class="col-span-7 flex items-end gap-10">
+			<img class="md:w-1/3 2xl:w-2/5 md:rounded-xl 2xl:rounded-3xl" src={anime.cardBackgroundImage} alt={anime.titles.eng} />
```

Comment:
This is not right

Width should be : `240px` 
Height should be : `360px`

Reply:
You said dont use arbitary values, or we need theming

Reply:
Yes

![image](https://user-images.githubusercontent.com/61817579/230700625-3097bf61-c48b-47d2-8175-60aab6a14ec3.png)

Reply:
Change width to `w-60`

Height will change automatically